CLOUDKITE.md

```diff
@@ -24,9 +24,9 @@ ssh-keygen -t rsa -b 4096 -f /mnt/volume/config/ssh_host_rsa_key < /dev/null
 ```bash
 mkdir -p /mnt/volume/scripts
 cp set_permissions.sh /mnt/volume/scripts
+cp create_home_dirs.sh /mnt/volume/scripts
```

Comment:
We should just move all of the custom scripts into a `startup-scripts` dir in this repo. Then you don't need custom instructions for each script.

Reply:
Fixed it in https://github.com/cloudkite-io/sftp/pull/7